crates/editor/src/display_map/block_map.rs

```diff
@@ -885,12 +885,12 @@ impl<'a> Iterator for BlockBufferRows<'a> {
 
     fn next(&mut self) -> Option<Self::Item> {
```

Comment:
In the same vein, should `BlockBufferRows::Item` be a `Option<BlockRow>` and not `Option<u32>`?